packages/react-native/React/Base/macOS/RCTUIKit.m

```diff
@@ -1040,4 +1003,40 @@ - (void)setImage:(UIImage *)image
 
 @end
 
+@implementation RCTUIGraphicsImageRendererFormat
+
++ (nonnull instancetype)defaultFormat {
+    static dispatch_once_t token = 0;
+    static RCTUIGraphicsImageRendererFormat *_sharedObject = nil;
+    dispatch_once(&token, ^{
+        _sharedObject = [[self alloc] init];
+    });
+    return _sharedObject;
+}
```

Comment:
Shouldn't this always return a new instance? It looks like it's being modified later, like in `RCTMakeUIGraphicsImageRenderer`.

Reply:
Right, good point. I think my brain forgot how to make a class method return an instance when I wrote it. Lets try again after a night of sleep :P